modules/apps/oauth2-provider/oauth2-provider-service/src/main/java/com/liferay/oauth2/provider/internal/upgrade/OAuth2ServiceUpgrade.java

```diff
@@ -73,7 +73,12 @@ public void register(Registry registry) {
 		registry.register(
 			"3.0.0", "4.0.0",
```

Comment:
Hi @martamedio,

Shouldn't be from 3.0.0 to 3.1.0? Also, please remember to change the field Liferay-Require-SchemaVersion in bnd.bnd

Thanks.

Reply:
Hey @achaparro, thanks for the heads up! 😃 

I'm keeping it in mind for when rebase from `master` (where it will already be: https://github.com/liferay-appsec/liferay-portal/pull/289)
This PR is still in process, so don't take this change into account at the moment, please, because all commits are previous to our conversation.

Reply:
Sorry @martamedio, I didn't realize about that pull.

Thanks for the update